### PR TITLE
cleaned up e2e tests

### PR DESCRIPTION
**Pull request by maheshKRex**
https://github.com/EmpowerPlastic/empowerchain/issues/559

### CODE REVIEW

File: chain/tests/e2e/plasticcredit/applicant_tx_test.go [hunk 1]
```diff
@@ -1,206 +0,0 @@
...
-			updatedTitle:      "Plastix Inc.",
-			updatedDesc:       "Grab that bottle",
-			fromFlagValue:     fmt.Sprintf("--%s=%s", flags.FlagFrom, issuerKey.Name),
-			expectedErrOnSend: false,
-			expectedErrOnExec: true,
-			expectedErrMsg:    "applicant not found",
-			expectedState:     nil,
-		},
-
-		"wrong signer": {
```
Comment (ba1ciu) [inline chain/tests/e2e/plasticcredit/applicant_tx_test.go:143]: We should add an integration test with a wrong signer, but one that will pass bech32 validation.

File: chain/tests/e2e/plasticcredit/issuer_tx_test.go [hunk 2]
```diff
@@ -10,256 +10,13 @@ import (
...
-	"github.com/cosmos/gogoproto/proto"
 
 	"github.com/EmpowerPlastic/empowerchain/app/params"
 	"github.com/EmpowerPlastic/empowerchain/tests/e2e"
-	"github.com/EmpowerPlastic/empowerchain/testutil/sample"
 	"github.com/EmpowerPlastic/empowerchain/x/plasticcredit"
 	"github.com/EmpowerPlastic/empowerchain/x/plasticcredit/client/cli"
 )
 
-func (s *E2ETestSuite) TestCmdCreateIssuer() {
```
Comment (ba1ciu) [inline chain/tests/e2e/plasticcredit/issuer_tx_test.go:22]: I'd leave issuer creation with gov. We don't have that covered on the integration level.

File: chain/x/plasticcredit/keeper/msg_server_test.go [hunk 3]
```diff
@@ -313,37 +303,41 @@ func (s *TestSuite) TestUpdateIssuer() {
 	}
 }
 
-func (s *TestSuite) TestCreateApplicant() {
+func (s *TestSuite) TestUpdateIssuerCreator() {
```
Comment (ba1ciu) [inline chain/x/plasticcredit/keeper/msg_server_test.go:306]: It's the same as `TestCreateIssuer`?

**Review by ba1ciu — CHANGES_REQUESTED** [review]

File: chain/tests/e2e/plasticcredit/applicant_tx_test.go (hunk 1 above)
Comment (maheshKRex) [reply chain/tests/e2e/plasticcredit/applicant_tx_test.go:143]: yes, having this in msg_server_tests, here sample.AccAddress generates some valid address, but not authorised to sign.
`"unauthorized caller / wrong signer": {
			msg: &plasticcredit.MsgUpdateApplicant{
				Updater:     sample.AccAddress(), 
				ApplicantId: 1,
				Name:        "Empower",
				Description: "Empower is cool",
				Admin:       issuerAdmin,
			},
			err: sdkerrors.Er`

File: chain/tests/e2e/plasticcredit/issuer_tx_test.go (hunk 2 above)
Comment (maheshKRex) [reply chain/tests/e2e/plasticcredit/issuer_tx_test.go]: agree. reverted the removal.

File: chain/x/plasticcredit/keeper/msg_server_test.go (hunk 3 above)
Comment (maheshKRex) [reply chain/x/plasticcredit/keeper/msg_server_test.go]: nice catch. removed them.

File: chain/tests/e2e/plasticcredit/issuer_tx_test.go [hunk 4]
```diff
@@ -10,13 +10,256 @@ import (
...
+				s.Require().NoError(val.ClientCtx.Codec.UnmarshalJSON(out.Bytes(), &resp))
+				s.Require().Equal(tc.expectedState.Name, resp.Issuers[len(resp.Issuers)-1].Name)
+				s.Require().Equal(tc.expectedState.Description, resp.Issuers[len(resp.Issuers)-1].Description)
+				s.Require().Equal(tc.expectedState.Admin, resp.Issuers[len(resp.Issuers)-1].Admin)
+			}
+		})
+	}
+}
+
+func (s *E2ETestSuite) TestCmdUpdateIssuer() {
```
Comment (ba1ciu) [inline chain/tests/e2e/plasticcredit/issuer_tx_test.go]: Updates we can get rid of I think as those are done by sending a standard message and leave only creates as they use `NewCmdSubmitProposal()`?

**Review by ba1ciu — CHANGES_REQUESTED** [review]

**Review by ba1ciu — APPROVED** [review]

**Pull request merged**